modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/frontend/data/set/provider/CommerceCheckoutDeliveryGroupFDSDataProvider.java

```diff
@@ -38,19 +38,19 @@
  * @author Luca Pellizzon
  */
 @Component(
-	property = "fds.data.provider.key=" + CommerceCheckoutFDSNames.DELIVERY_GROUP,
+	property = "fds.data.provider.key=" + CommerceCheckoutFDSNames.DELIVERY_GROUPS,
```

Comment:
if now this is plural is it correct that the class is singular CommerceCheckout**DeliveryGroup**FDSDataProvider? Also other classes has the name singular